modules/overrides/config.go

```diff
@@ -194,7 +199,8 @@ type Overrides struct {
 	// Global enforced overrides.
 	Global GlobalOverrides `yaml:"global,omitempty" json:"global,omitempty"`
 	// Storage enforced overrides.
-	Storage StorageOverrides `yaml:"storage,omitempty" json:"storage,omitempty"`
+	Storage         StorageOverrides         `yaml:"storage,omitempty" json:"storage,omitempty"`
+	CostAttribution CostAttributionOverrides `yaml:"cost_attribution,omitempty" json:"cost_attribution,omitempty"`
```

Comment:
we have been calling this `usage tracker` or `usage metrics` at other places but Cost Attribution in the overrides. 

let's use `usage tracker` or `usage metrics` here to keep it consistent and avoid confusion.

Reply:
ah, please ignore my comments then.